jac-uk/admin
# #1621 character issues - reason for recommendation

**Author checklist**


---
## What's included?
- [x] Added a Reason for Recommendation field to the Character Issues report.
- [x] Reason for Recommendation field should become visible for each character application once a status has been chosen

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
Preview URL: https://jac-admin-develop--pr1623-feature-1621-reason-p7lxd9fs.web.app
- Open an exercise with character issues in it
- Check that the Reason for Recommendation field is hidden if a status has not been chosen for an application yet
- Check that the Reason for Recommendation field is visible once a status has been chosen for an application
- Check that data is saved when you enter text in the Reason for Recommendation field 

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Demo:

https://user-images.githubusercontent.com/79906532/171233747-e3a1e992-d061-450d-b75e-9f10afddd567.mov

## Related permissions
Have permissions been considered for this functionality?
- No permission changes required

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
